src/components/Calculator.css

```diff
@@ -1,4 +1,33 @@
+
 .calculator {
-  max-width: 500px;
-  margin: 10px auto;
+  margin-top: 0;
+  width: 100%;
+  border-collapse: collapse;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.calculator button {
+  border: 1px #777 solid;
+  padding: 50px;
+  font-weight: 700;
+  font-size: 18px;
+  cursor: pointer;
+
+  /* width: 125px; */
+}
```

Comment:
- [ ] Kindly remove commented unused code from your codebase as this makes your code not clean and professional. Please remove all used commented codes 👍🏽